demos/node-classification/deep-graph-info-max/gcn-deep-graph-info-max-cora-example.ipynb

```diff
@@ -187,7 +187,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Test classification accuracy: 0.7805578342904019\n"
+      "Test classification accuracy: 0.7358490566037735\n"
```

Comment:
This dropped significantly with this run. Is the variance very high? It looks like we're still significantly below the paper, and higher variance than it?

Reply:
Performance variance with train/test/val splits is very high. The paper has a custom cora data set that's pre-split so I was only able to replicate it by using their data loaders, I have a notebook for this if you're interested and running this gets consistent results around 81%.

Reply:
Ah, I didn't realise you were doing that in a separate notebook. If we're not going to include in this notebook (which is fine), we should at least include a comment/piece of text somewhere that mentions the difference.